pkg/controllers/workspace_controller.go

```diff
@@ -37,10 +38,26 @@ func (c *WorkspaceReconciler) Reconcile(ctx context.Context, req reconcile.Reque
 		}
 		return reconcile.Result{}, client.IgnoreNotFound(err)
 	}
+
 	klog.InfoS("Reconciling", "workspace", req.NamespacedName)
+	if !apimeta.IsStatusConditionFalse(workspaceObj.Status.Conditions, string(kdmv1alpha1.WorkspaceConditionTypeReady)) {
```

Comment:
Let us remove it for now, You cannot make condition not ready whenever runs the reconciler. Remember, it can be noop for a reconcile.

Reply:
Removed.